Cargo.lock

```diff
@@ -3410,3 +3410,11 @@ checksum = "ec866b44a2a1fd6133d363f073ca1b179f438f99e7e5bfb1e33f7181facfe448"
 dependencies = [
  "zune-core",
 ]
+
```

Comment:
Why do you keep trying to make this change?
We've already had to fix this once in #679

Reply:
I don't know really, i don't touch Cargo.lock so it's probably changed when i run clippy.

When i was working on the SDL2 branch, i had this issue of Cargo.lock being constantly "outdated" when i ran the tests, and the fix was to run clippy, which it'll change the file.

I'm sorry if that caused issues, i'll see if i can fix that on my end.

Reply:
Oh, that's a bit annoying. Thanks for sorting it.

Reply:
No problem! :D